cmd/skopeo/tags.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/containers/image/v5/docker"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+
+	"io"
+)
+
+// tagListOutput is the output format of (skopeo tags), primarily so that we can format it with a simple json.MarshalIndent.
+type tagListOutput struct {
+	DockerRepository string   `json:",omitempty"`
+	Tags             []string `json:",omitempty"`
+}
+
+type tagsOptions struct {
+	global *globalOptions
+	image  *imageOptions
+}
+
+func tagsCmd(global *globalOptions) cli.Command {
+	sharedFlags, sharedOpts := sharedImageFlags()
+	imageFlags, imageOpts := imageFlags(global, sharedOpts, "", "")
```

Comment:
There is now a `dockerImageFlags()`, which drops some irrelevant options (which the documentation was already silentl about).

Reply:
ack, will update here